documentation/src/main/asciidoc/topics/ref_persistence_spi.adoc

```diff
@@ -0,0 +1,48 @@
+= Persistence SPI Classes
+
+The following class diagram shows the main interfaces of the persistence SPI:
+
+image::{images_dir}/Figure2_1_persistence_API.png[align="center", title="Persistence SPI"]
+
+.Notes about the classes
+
+link:{javadocroot}/org/infinispan/commons/io/ByteBuffer.html[ByteBuffer]:: Abstracts the serialized form of an object.
+
+link:{javadocroot}/org/infinispan/persistence/spi/MarshalledEntry.html[MarshalledEntry]::
```

Comment:
This is deprecated. We should mention `MarshallableEntry` instead. The description is still appropriate though.